test/gtest/cfg.cpp

```diff
@@ -579,3 +581,67 @@ TEST_F(CFGTest, ReachingDefinitionsLoop) {
 
   EXPECT_EQ(expectedResult, getSetses);
 }
+
+TEST_F(CFGTest, StackBitsLatticeFunctioning) {
+  BitsLattice bitsLattice;
+  StackLattice<BitsLattice> stackLattice(bitsLattice);
```

Comment:
Can we test these two separately? Or are they only meant to be used with each other? If so that wasn't obvious to me from the headers.

Reply:
The `StackLattice` needs another lattice to model stack contents (i.e. each stack element). However, I will replace `BitsLattice` with an existing lattice.